app/mailers/admin_mailer.rb

```diff
@@ -26,6 +26,15 @@ def email_stats(hours)
     mail from: SENDING_EMAIL_ADDRESS, to: ADMIN_EMAILS, subject: "FromThePage activity in the last #{hours} hours."
   end
 
+  def collection_stats_by_owner(owner, start_date, end_date)
+    @collections = owner.all_owner_collections.joins(:deeds).where(deeds: {created_at: start_date..end_date})
+    @start_date = start_date
+    @end_date = end_date
+    
+    mail from: SENDING_EMAIL_ADDRESS, to: owner.email, subject: "FromThePage collection activity in the last 24 hours"
```

Comment:
Ah, maybe I misunderstand.  One email with all collections listed, similar to the admin email?